tests/test_line.py

```diff
@@ -0,0 +1,121 @@
+import unittest
+from ai_graph_color.line import Line, point_distance
+
+
+class TestLineObject(unittest.TestCase):
+    def test_init_unequal_x(self):
+        """
+        Tests making a line with unequal x values
```

Comment:
They are different. One has x values that are unequal and one has y values that are unequal